be/src/storage/rowset/bloom_filter.h

```diff
@@ -50,6 +51,37 @@ class Slice;
 static const std::string FPP_KEY = "bloom_filter_fpp";
 static const std::string GRAM_NUM_KEY = "gram_num";
 static const std::string CASE_SENSITIVE_KEY = "case_sensitive";
+
```

Comment:
It's better to move these entire implementation to a shared directory instead of still sitting in `be/src/storage`